### DISCUSSION TITLE
A public alternative to Discord?

### DISCUSSION THREAD

**Discussion by tcw6ecv** [LeelaChessZero/lc0#1745] [Q&A]
Is there a public alternative to our discord server?

### What I mean by `public`?

The problem is that discord requires a mobile phone verification:
![image](https://user-images.githubusercontent.com/104824875/166422938-b9f7e90c-a1bc-41ea-b845-1c709b09c16b.png)

What is even worse is that after I provided my phone number, my account was disabled for "Violation of TOS/Community Guidelines" (do not ask me how a new account could violate anything in the first place).

### Why I need discord?

I am one of top 50 overall contributors [here](https://training.lczero.org/) (moreover, I have just bought a 2nd GPU), and it would very nice to have at least read-only access to discord / notifications / announcements...

So, is there a way around?

@Tilps

**Comment by mooskagh** [comment]
While it's true that many Discord servers require a phone number verification, this is not enabled for our server.
Our server only requires email address, and that's the lowest verification level that is allowed to set for a public server.

It is possible that they still ask it from IP addresses that are know to be public VPNs or something like that.

If you believe that you did not violate Discord TOS, you can file an appeal here: https://support.discord.com/hc/en-us/requests/new (under "Trust & Safety" -> "Appeals"). We did have successful stories of going through this process (but it was a clear case, posts in #tests-results were misclassified as spam).

**Comment by sroller** [comment]
You could try Zulip which just announced the option for public access.
https://zulip.com/why-zulip/

**Reply by mooskagh** [reply]
While I agree that the problem listed in /why-zulip are typical for many discord/slack servers, for Lc0 server particularly none of the problems are true:

- We don't have busy channels,
- All core devs read all messages through,
- We don't have any problems with gifs/meme posting,
- The most of the communication does happen in Discord, and not in emails, direct messages etc.

In general, it does feel that zulip would be a great too for many use cases, but for Lc0 it would bring more problems (bringing people to less popular service) than good.
